resources/sass/style.scss

```
@@ -1,3 +1,7 @@
 #countries {
     width: 300px;
 }
+
+logout_form {
```

```suggestion
.logout_form {
```
class hay id đây em?